cmd/functions.go

```diff
@@ -42,7 +42,7 @@ var (
 		},
 	}
 
-	noVerifyJWT     bool
+	noVerifyJWT     = new(bool)
```

Comment:
Isn't this unnecessary? because users can never pass in `noVerifyJWT = false` explicitly via cli flag. So if the flag is false, we can safely default to load from config file.

Reply:
ok I guess they could do `--no-verify-jwt=false`... but it's kind of a rare case because `--no-verify-jwt false` will throw an error.

```bash
$ supabase functions deploy hello_world --no-verify-jwt false
Error: accepts 1 arg(s), received 2
```

Reply:
That sounds unintuitive - flags should always override config file if specified. Alternatively we could throw an error if a user does `--no-verify-jwt=false` and ask them to drop the flag.

Reply:
Seems like having boolean flags accept true/false args has caused confusion before with cobra users https://github.com/spf13/cobra/issues/1657.

```bash
$ man --ascii=true
man: option '--ascii' doesn't allow an argument
```

I'd prefer rejecting `--no-verify-jwt=false` for clarity coz it's double negation. But may be we can improve handling in a separate PR that deals with all our boolean flags.

Reply:
Yeah, I'd prefer disallowing arguments to bool flags altogether, but alas...